Content.Server/Doors/Systems/AirlockSystem.cs

```diff
@@ -149,8 +151,11 @@ private void OnBeforeDoorDenied(EntityUid uid, AirlockComponent component, Befor
 
     private void OnActivate(EntityUid uid, AirlockComponent component, ActivateInWorldEvent args)
     {
-        if (TryComp<WiresPanelComponent>(uid, out var panel) && panel.Open && panel.WiresAccessible
-            && TryComp<ActorComponent>(args.User, out var actor))
+        if (TryComp<WiresPanelComponent>(uid, out var panel) &&
+            panel.Open &&
+            _prototypeManager.TryIndex<WiresPanelSecurityLevelPrototype>(panel.CurrentSecurityLevelID, out var securityLevelPrototype) &&
```

Comment:
Why is this from the prototype instead of from the component's current state?

Reply:
Because Sloth asked that the revelant data be read from the prototype isn't of having duplicated fields in the component